Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/MeshComponentController.cpp

```diff
@@ -85,6 +86,13 @@ namespace AZ
             return values;
         }
 
+        AZStd::vector<AZStd::pair<RPI::Cullable::LodType, AZStd::string>> MeshComponentConfig::GetLodTypeValues()
+        {
+            return {
+                {RPI::Cullable::DefaultLodType, "Default"}
+            };
+        }
+
```

Comment:
Can make this the 3 types suggested above, using an enum defined in the MeshComponentConfig that matches the one from Culling.